src/inmanta/agent/handler.py

```diff
@@ -909,7 +909,7 @@ def execute(self, ctx: HandlerContext, resource: resources.Resource, dry_run: Op
                     changes["purged"] = dict(desired=desired.purged, current=True)
 
             for field, values in changes.items():
-                ctx.add_change(field, desired=values["desired"], current=values["current"])
+                ctx.add_change(field, desired=changes[field]["desired"], current=changes[field]["current"])
```

Comment:
Why was this changed?

Reply:
This is an artifact from a previous iteration, reverting the change now.